indigo_app/templates/indigo_api/_task_detail.html

```diff
@@ -260,12 +260,14 @@ <h6>Labels</h6>
         {% endwith %}
 
         <form action="{% url 'task_blocked_by' place=place.place_code pk=task.pk %}" method="POST"
-              id="task-blocked_by-form" onchange="this.submit();">
+              id="task-blocked_by-form">
           {% csrf_token %}
 
-          <select class="selectpicker" multiple name="blocked_by"
+          <select class="selectpicker" multiple name="blocked_by" onchange="$(this.form).trigger('submit')"
```

Comment:
Any reason to use `trigger('submit')` vs calling `this.form.submit()`?

Reply:
no good reason; I was just copying what we had before :bow: